qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -675,6 +675,76 @@ def test_df_for_valid_file(self):
         self.validate_df("dumpfile")
 
 
+class TestQuota(TestCephFSShell):
+    dir_name = 'testdir'
+
+    def create_dir(self):
+        mount_output = self.get_cephfs_shell_cmd_output('mkdir ' + self.dir_name)
+        log.info("cephfs-shell mount output:\n{}".format(mount_output))
+
+    def validate(self, input_val):
```

Comment:
While reading tests, it's not very discernible what this method does from it's name. Would be better if you could use a more specific name, especially from POV of someone who'll debug this method in future.

Reply:
I have renamed the function.